03-task-manager/starter/errors/custom-error.js

```diff
@@ -0,0 +1,13 @@
+
+class CustomAPIError extends Error {
+    constructor (message, statusCode) {
+        super(message)
+        this.statusCode = statusCode
+    }
+}
+
+const createCustomError = (message, statusCode) => {
+    return newCustomAPIError(message, statusCode)
```

Comment:
You should instantiate the CustomAPIError here. The newCustomAPIError method doesn't exist.
Ex: return new CustomAPIError(message, statusCode)